### ISSUE TITLE
Immediate changes, esp. for pre-hackweek materials

### ISSUE THREAD

**Issue by emiliom** [oceanhackweek/ohw-resources#3]
@leewujung can you list out the pre-hackweek content / pages you want to see, specially those involving porting the content from https://oceanhackweek.github.io/wiki/index.html ?

All the content documents are in the [docs](https://github.com/oceanhackweek/ohw-learning-resources/tree/master/docs) folder. The organization is defined in [mkdocs.yml](https://github.com/oceanhackweek/ohw-learning-resources/blob/master/mkdocs.yml), in its `nav` section.

Looking at the old wiki and other materials from 2019, I can identify these documents:
- **Wiki**
  - Overview
    - https://oceanhackweek.github.io/wiki/index.html
    - https://oceanhackweek.github.io/wiki/code_of_conduct.html
    - https://oceanhackweek.github.io/wiki/mission_statement.html
  - Pre-Hackweek Preparation
    - https://oceanhackweek.github.io/wiki/python_requirements.html
    - https://oceanhackweek.github.io/wiki/github_setup.html
- **Other ohw19 repos**
  - https://github.com/oceanhackweek/preliminary. Built as the carpentry-template site https://oceanhackweek.github.io/preliminary/ with two episodes
  - https://github.com/oceanhackweek/ohw19-tutorial-overview. Two notebooks and one pdf presentation
  - https://github.com/oceanhackweek/ohw19-tutorial-github (note that I folded much of this content into https://oceanhackweek.github.io/ohw-learning-resources/git/)

I've also compiled a number of other resources from other hackweeks and other sources, which I can add once we settle on a structure.

**Comment by leewujung** [comment]
Sure, I'll do that after the allhands meeting today so that I know what preweek materials should be there.

**Comment by emiliom** [comment]
I've looked at the ICESat-2 hackweek learning resources site more closely. I like the first-order organization of topics, and recommend we use it for OHW:

- Overview
- Conduct
- Logistics
- Preliminary Steps
- Tutorials
- Projects
- Reference
- About

I think we can trim some of those -- for example, dropping Reference and About as distinct first-order topics. We can also rename some of the others; Preliminary Steps could be Pre-Hackweek, for example.

I also like how they organized the markdown files (see [here](https://github.com/ICESAT-2HackWeek/learning-resources/blob/master/mkdocs.yml#L2)). I think it makes it easier to see what's what, compared to a flat folder structure.

**Comment by leewujung** [comment]
> I also like how they organized the markdown files (see here). I think it makes it easier to see what's what, compared to a flat folder structure.

Yes that's fantastic! Much easier to use than the Jekyll theme structure -- that one seems too bootstrap for us.

I'll take a stab at trimming and changing these and do a PR, and we can iterate from there. ;)

**Comment by leewujung** [comment]
@emiliom : I'm only half way done [here](https://github.com/leewujung/ohw-learning-resources/tree/first-trim/docs) but let me know if you have any comments and I can work them in tomorrow evening.

**Comment by emiliom** [comment]
Looking great, @leewujung ! Please just submit a PR as is and ping me. I'll review it and merge; actually, I've already reviewed it -- I cloned your branch and tested it locally.

Since you've made large organizational changes, it'll be better to have this PR in place, to then add to them. I have a style change I'll apply, from the ICESat-2 hackweek site, plus a bit of reorganization. After that, changes will be more incremental and focused on content, as it should be.

**Comment by emiliom** [comment]
FYI, @leewujung's changes were merged (#4), and then I deployed the markdown file reorganization into logical subdirectories (#6). I plan to work on the site further today.

Wu-Jung's addition of the GH Action to auto-build the site (`gh_pages`) is now online!!
